src/routes/conversation/[id]/+server.ts

```diff
@@ -101,10 +102,17 @@ export async function POST({ request, locals, params, getClientAddress }) {
 		await collections.messageEvents.countDocuments({ ip: getClientAddress() })
 	);
 
-	if (RATE_LIMIT != "" && nEvents > parseInt(RATE_LIMIT)) {
+	if (RateLimits?.messagesPerMinute && nEvents > RateLimits.messagesPerMinute) {
```

Comment:
Should we only fetch `nEvents` if `RateLimits?.messagePerMinute` is defined?

Reply:
yes indeed, fixed